projects/dotcms-field-elements/src/components/dot-binary-file/dot-binary-text-field/dot-binary-text-field.tsx

```diff
@@ -0,0 +1,182 @@
+import { Component, Element, Event, EventEmitter, Method, Prop, State, Watch } from '@stencil/core';
+import Fragment from 'stencil-fragment';
+import { DotBinaryTextStatusEvent, DotFieldStatus, DotFieldValueEvent } from '../../../models';
+import {
+    getErrorClass,
+    getOriginalStatus,
+    isFileAllowed,
+    isValidURL,
+    updateStatus
+} from '../../../utils';
+
+/**
+ * Represent a dotcms text field for the binary file element.
+ *
+ * @export
+ * @class DotBinaryFile
+ */
+@Component({
+    tag: 'dot-binary-text-field',
+    styleUrl: 'dot-binary-text-field.scss'
+})
+export class DotBinaryTextFieldComponent {
+    @Element() el: HTMLElement;
+
+    /** Value specifies the value of the <input> element */
+    @Prop({ mutable: true, reflectToAttr: true })
+    value = null;
+
+    /** (optional) Placeholder specifies a short hint that describes the expected value of the input field */
+    @Prop({ reflectToAttr: true })
+    placeholder = 'Attach files by dragging & dropping, selecting or pasting them.';
+
+    /** (optional) Determine if it is mandatory */
+    @Prop({ reflectToAttr: true })
+    required = false;
+
+    /** (optional) Array that describes a type of file that may be selected by the user, eg: .pdf,.jpg  */
+    @Prop({ reflectToAttr: true })
+    accept = '';
```

Comment:
`filesTypeAccepted` better name?

Reply:
name is based in the standard attribute of an `input`